src/sections/strategy.html

```diff
@@ -0,0 +1,25 @@
+<div class="lg:px-36 px-10 md:px-20 relative">
+    <div class="w-full flex justify-center sticky top-0 z-20">
+        <img src="/images/small-yellow-shape.svg" alt="" class=" lg:w-[18%]  w-[60%] md:w-[25%]">
+    </div>
+    <div class="grid lg:grid-cols-2 lg:gap-60 pt-32 grid-rows-1 gap-20 md:gap-20">
+          <div class=" text-3xl lg:text-4xl md:text-4xl font-medium  flex flex-col lg:gap-16 gap-12">
```

Comment:
avoid unnecessary spaces in your codebase
```suggestion
          <div class="text-3xl lg:text-4xl md:text-4xl font-medium flex flex-col lg:gap-16 gap-12">
```